tests/test_command.py

```
@@ -2348,12 +2348,33 @@ def test__show_cache_cmd(self):
+        self.assertEqual(cm.exception.code, 2)
+
+    def test_boom_main_bad_type(self):
+        args = ["bin/boom", "quux", "list"]
+        with self.assertRaises(SystemExit) as cm:
+            status = boom.command.main(args)
+        self.assertEqual(cm.exception.code, 2)
+
+    def test_boom_main_bad_command(self):
+        args = ["bin/boom", "entry", "quux"]
+        with self.assertRaises(SystemExit) as cm:
+            status = boom.command.main(args)
```

Ruff (0.12.2)

Local variable `status` is assigned to but never used

Remove assignment to unused variable `status`

(F841)